src/tests/functional/plugin/cpu/subgraph_tests/src/input_tensor_roi.cpp

```diff
@@ -11,6 +11,8 @@ using namespace ngraph::op;
 using namespace InferenceEngine;
 using namespace CPUTestUtils;
 
+namespace CPULayerTestsDefinitions {
```

Comment:
This is a subgraph test so may be we should use `CPUSubgraphTestsDefinitions` namespace here?

Reply:
renamed `CPULayerTestsDefinitions ` -> `SubgraphTestsDefinitions`